From penberg@cs.helsinki.fi  Wed Jan 21 09:28:46 2009
From: Pekka J Enberg <penberg@cs.helsinki.fi>
Date: Wed, 7 Jan 2009 17:32:58 +0200 (EET)
Subject: Staging: w35und: remove unnecessary os_common.h header file
To: Greg Kroah-Hartman <greg@kroah.com>
Cc: Pavel Machek <pavel@suse.cz>
Message-ID: <Pine.LNX.4.64.0901071732410.17909@melkki.cs.Helsinki.FI>


From: Pekka Enberg <penberg@cs.helsinki.fi>

Convert code to include sysdef.h directly and remove unnecessary os_common.h
header file.

Signed-off-by: Pekka Enberg <penberg@cs.helsinki.fi>
Acked-by: Pavel Machek <pavel@suse.cz>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/winbond/mds.c             |    2 +-
 drivers/staging/winbond/mlmetxrx.c        |    2 +-
 drivers/staging/winbond/mto.c             |    2 +-
 drivers/staging/winbond/os_common.h       |    2 --
 drivers/staging/winbond/phy_calibration.c |    2 +-
 drivers/staging/winbond/reg.c             |    2 +-
 drivers/staging/winbond/wbhal.c           |    2 +-
 7 files changed, 6 insertions(+), 8 deletions(-)

--- a/drivers/staging/winbond/mds.c
+++ b/drivers/staging/winbond/mds.c
@@ -3,7 +3,7 @@
 #include "mds_f.h"
 #include "mlmetxrx_f.h"
 #include "mto_f.h"
-#include "os_common.h"
+#include "sysdef.h"
 #include "wbhal_f.h"
 #include "wblinux_f.h"
 
--- a/drivers/staging/winbond/mlmetxrx.c
+++ b/drivers/staging/winbond/mlmetxrx.c
@@ -15,7 +15,7 @@
 //
 //  Copyright (c) 1996-2002 Winbond Electronics Corp. All Rights Reserved.
 //============================================================================
-#include "os_common.h"
+#include "sysdef.h"
 
 #include "mds_f.h"
 
--- a/drivers/staging/winbond/mto.c
+++ b/drivers/staging/winbond/mto.c
@@ -22,7 +22,7 @@
 //============================================================================
 
 // LA20040210_DTO kevin
-#include "os_common.h"
+#include "sysdef.h"
 #include "sme_api.h"
 #include "gl_80211.h"
 #include "wbhal_f.h"
--- a/drivers/staging/winbond/os_common.h
+++ /dev/null
@@ -1,2 +0,0 @@
-#include "sysdef.h"
-
--- a/drivers/staging/winbond/phy_calibration.c
+++ b/drivers/staging/winbond/phy_calibration.c
@@ -10,7 +10,7 @@
  */
 
 /****************** INCLUDE FILES SECTION ***********************************/
-#include "os_common.h"
+#include "sysdef.h"
 #include "phy_calibration.h"
 #include "wbhal_f.h"
 
--- a/drivers/staging/winbond/reg.c
+++ b/drivers/staging/winbond/reg.c
@@ -1,4 +1,4 @@
-#include "os_common.h"
+#include "sysdef.h"
 #include "wbhal_f.h"
 
 ///////////////////////////////////////////////////////////////////////////////////////////////////
--- a/drivers/staging/winbond/wbhal.c
+++ b/drivers/staging/winbond/wbhal.c
@@ -1,4 +1,4 @@
-#include "os_common.h"
+#include "sysdef.h"
 #include "wbhal_f.h"
 #include "wblinux_f.h"
 
